<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.41">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>schedule – European Bioconductor Conference 2025</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../images/favicon.ico" rel="icon">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-48ffa3e5b9d089919c6712c39e5b00f2.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap-e6ddf4e3e105cc8a8b72f24854d1dacb.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<link rel="stylesheet" href="../styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../images/logoBioc.svg" alt="EuroBioC2025" class="navbar-logo">
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link active" href="../pages/schedule.html" aria-current="page"> 
<span class="menu-text">Schedule</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../pages/speakers.html"> 
<span class="menu-text">Speakers</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../pages/submissions.html"> 
<span class="menu-text">Submissions</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../pages/registration.html"> 
<span class="menu-text">Registration</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../pages/carpentryworkshops.html"> 
<span class="menu-text">Carpentry-workshops</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../pages/sponsors.html"> 
<span class="menu-text">Sponsors</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-about" role="link" data-bs-toggle="dropdown" aria-expanded="false">
 <span class="menu-text">About</span>
    </a>
    <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="nav-menu-about">    
        <li>
    <a class="dropdown-item" href="../pages/organizers.html">
 <span class="dropdown-text">Organizers</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../pages/past-events.html">
 <span class="dropdown-text">Past Events</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../pages/code-of-conduct.html">
 <span class="dropdown-text">Code of Conduct</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../pages/travel-information.html">
 <span class="dropdown-text">Travel Information</span></a>
  </li>  
    </ul>
  </li>
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#schedule" id="toc-schedule" class="nav-link active" data-scroll-target="#schedule">Schedule</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content"><header id="title-block-header" class="quarto-title-block"></header>




<section id="schedule" class="level1">
<h1>Schedule</h1>
<table class=" lightable-material" style="font-family: &quot;Source Sans Pro&quot;, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
time
</th>
<th style="text-align:left;">
type
</th>
<th style="text-align:left;">
author
</th>
<th style="text-align:left;">
title
</th>
</tr>
</thead>
<tbody>
<tr grouplength="25">
<td colspan="4" style="border-bottom: 1px solid #eee; ">
<strong>Wed. - Sep.&nbsp;17, ’25</strong>
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
9:00
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Opening
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Organizing committee
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
Welcome to EuroBioC
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; background-color: lavender !important;padding-left: 2em;border-bottom: 1px solid lightyellow" indentlevel="1">
9:15
</td>
<td style="text-align:left;width: 20%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Keynote
</td>
<td style="text-align:left;width: 25%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Vince Carey
</td>
<td style="text-align:left;width: 40%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
A coherent ecosystem for genomic data science: 25 years of Bioconductor
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; background-color: lavender !important;padding-left: 2em;border-bottom: 1px solid lightyellow" indentlevel="1">
9:45
</td>
<td style="text-align:left;width: 20%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Keynote
</td>
<td style="text-align:left;width: 25%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Helena Crowell
</td>
<td style="text-align:left;width: 40%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Colorectal cancer through the lens of whole transcriptome imaging
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; background-color: lightyellow !important;padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
10:15
</td>
<td style="text-align:left;width: 20%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
Break
</td>
<td style="text-align:left;width: 25%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
11:00
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Short talks
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Vilhelm Suksi
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="notame: R/Bioconductor package for untargeted LC–MS metabolomics data analysis" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-31-contents" aria-controls="callout-31" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
notame: R/Bioconductor package for untargeted LC–MS metabolomics data analysis
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-31" class="callout-31-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>There is increasing interest in the interplay of metabolism, pathologies and biological systems such as the immune system and the gut microbiome. Untargeted liquid chromatography-mass spectrometry (LC-MS) attracts many new practitioners in quantitative metabolomics research, largely due to its sensitivity and broad coverage of the metabolome: the small molecules in a biological sample. However, due to experimental reasons and the extensive data analysis, untargeted LC-MS metabolomics data analysis meets challenges with regard to quality and reproducibility. In an effort to meet these challenges, the notame R package was developed in parallel with an associated protocol article published in the “Metabolomics Data Processing and Data Analysis—Current Best Practices” special issue of the Metabolites journal. The focus is on a solid starting point for new practitioners. The main outcome is identifying interesting features for laborious downstream steps relating to biological context, such as metabolite identification and pathway analysis, which fall outside the purview of notame. To further promote quality and reproducibility, notame has now been substantially upgraded to be included in Bioconductor, a repository focused on high-quality open software for omics research.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Philippine Louail
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Metabonaut: A collection of tutorials to learn metabolomics data analysis in R." style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-30-contents" aria-controls="callout-30" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Metabonaut: A collection of tutorials to learn metabolomics data analysis in R.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-30" class="callout-30-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Untargeted LC-MS/MS is a powerful approach for large-scale metabolomics studies, yet reproducible and efficient analysis of such data remains a major challenge. While R offers highly customizable workflows suited to diverse experimental and instrumental setups, the integration of specialized packages into coherent, scalable pipelines—especially for large cohort analyses—is often complex and fragmented. To address this gap, we present Metabonaut, an educational resource comprising a series of reproducible tutorials for untargeted LC-MS/MS metabolomics data analysis using R and Bioconductor. Built around a representative LC-MS/MS dataset, the tutorials demonstrate how to construct an end-to-end analysis workflow using tools such as xcms and other packages from the RforMassSpectrometry ecosystem. Each tutorial guides users step-by-step through the analysis process—from raw data preprocessing and feature detection to statistical analysis and annotation—emphasizing reproducibility, adaptability, and interoperability. As a case study, we include an analysis of human plasma samples comparing individuals with cardiovascular disease to healthy controls, illustrating quality control, normalization, and differential abundance analysis. Beyond core workflows, Metabonaut offers modules on data inspection and quality assessment, flexible alignment for integrating new data into existing preprocessed sets, and cross-language interoperability—highlighted through spectral annotation using Python’s matchms library. All tutorials are designed to be executable over time and can be used independently or combined into a comprehensive “super-vignette.” This work is supported by the European Union under the HORIZON-MSCA-2021 project 101073062: HUMAN – Harmonising and Unifying Blood Metabolic Analysis Networks.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Alexandre Segers
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="omicsGMF: dimensionality reduction, batch correction and imputation of missing values for proteomics data" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-29-contents" aria-controls="callout-29" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
omicsGMF: dimensionality reduction, batch correction and imputation of missing values for proteomics data
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-29" class="callout-29-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Technical advancements in mass spectrometry have enabled large-scale proteomics studies and even proteome profiling at the single cell level. However, the huge number of missing values and technical batch effects result in challenges for data analysis. This hinders a first key step in the data analysis workflow, i.e., dimensionality reduction, important for data exploration, visualization, and QC, as well as for downstream applications such as clustering cells. To this end, chained workflows are currently used that sequentially impute missing values, correct for batch effects, and perform conventional principal component analysis. However, their results depend on the order of the tools used in the workflow. Moreover, missingness is influenced by batch effects, so there is no guarantee that chained workflows lead to optimal, interpretable, or reliable results. We present omicsGMF, the first package for proteomics that integrates dimensionality reduction, batch correction and missing value imputation within a single framework. It builds on our sgdGMF framework, which uses a stochastic gradient descent for generalized matrix factorization and was previously used for dimensionality reduction on single cell RNA-sequencing data. omicsGMF is optimized for omics data and is here applied to a diverse set of bulk and single-cell proteomics data. omicsGMF can be easily used on SummarizedExperiment, SingleCellExperiment and QFeatures classes, and can model the data using different members of the exponential family, such as Gaussian, Poisson or negative binomial distributions. In this contribution we first show that omicsGMF improves the quality of dimensionality reduction and visualization, while simultaneously addressing batch effect removal and missing values. Second, omicsGMF can assist the user to select the optimal dimensionality by cross-validation, which improves downstream analysis. Furthermore, we illustrate how it can be used for imputation of missing values, resulting in superior imputation performance than state-of-the-art imputation tools, which in turn leads to superior sensitivity and specificity in downstream differential abundance analyses. By providing an all-in-one solution for dimensionality reduction, batch correction and imputation that is highly interpretable, omicsGMF addresses a critical gap in proteomics data analysis for single cell and large-scale applications.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Geraldson T. Muluh
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="miaTime: A scalable R/Bioconductor framework for longitudinal microbiome analysis" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-28-contents" aria-controls="callout-28" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
miaTime: A scalable R/Bioconductor framework for longitudinal microbiome analysis
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-28" class="callout-28-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Understanding how microbial communities change over time is essential for studying ecological stability, disease progression, and treatment response. Longitudinal microbiome studies enable these insights but pose analytical challenges, including repeated measures, missing data, batch effects, and the complexity of temporal modeling. miaTime is a new R/Bioconductor package designed to directly address these challenges. miaTime extends the mia (Microbiome Analysis) framework with a standardized, scalable approach for managing, preprocessing, and analyzing temporal microbiome data. Like other mia tools, it builds on TreeSummarizedExperiment (TreeSE), an advanced data structure for storing microbiome feature tables linked to phylogenetic trees. The package implements a suite of analysis methods tailored for time-series microbiome studies, including measures of community stability, detection of bimodal abundance patterns, identification of short-term changes, and quantification of stepwise and baseline divergence. Designed for full interoperability within the R/Bioconductor ecosystem, miaTime advances longitudinal microbiome research across clinical, environmental, and systems biology applications.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Johannes Rainer
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="SpectriPy: Enhancing Cross-Language Mass Spectrometry Data Analysis with R and Python" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-27-contents" aria-controls="callout-27" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
SpectriPy: Enhancing Cross-Language Mass Spectrometry Data Analysis with R and Python
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-27" class="callout-27-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Mass spectrometry (MS) is a key technology used across multiple fields, including biomedical research and life sciences. Technological advancements result in increasingly large and complex data sets and analyses must be tailored to the experimental and instrumental setups. Excellent software libraries for such data analysis are available in both R and Python, including R packages from the RforMassSpectrometry initiative and Python libraries like <em>matchms</em>, <em>spectrum_utils</em>, <em>Pyteomics</em> and <em>pyOpenMS</em>. Having partially complimentary functionality, these software cover different aspects of MS-based proteomics or metabolomics data analysis. The <em>reticulate</em> R package provides an R interface to Python enabling interoperability between the two programming languages. Here we present the <em>SpectriPy</em> R package that builds upon <em>reticulate</em> and provides functionality to efficiently translate between R and Python MS data structures. It can convert between R’s <code>Spectra::Spectra</code> and Python’s <code>matchms.Spectrum</code> and <code>spectrum_utils.spectrum.MsmsSpectrum</code> objects and includes functionality to directly apply spectral similarity, filtering, normalization, etc. routines from the Python <em>matchms</em> library on MS data in R. <em>SpectriPy</em> hence enables and simplifies the integration of R and Python for MS data analysis, empowering data analysts to benefit from the full power of algorithms in both programming languages. Furthermore, software developers can reuse algorithms across languages rather than re-implementing them, enhancing efficiency and collaboration.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Florian Auer
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="RCX2 - a package adapting the subsequent development of the Cytoscape Exchange format for biological networks" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-26-contents" aria-controls="callout-26" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
RCX2 - a package adapting the subsequent development of the Cytoscape Exchange format for biological networks
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-26" class="callout-26-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>RCX2 is an R package implementing the CX2 specification, the next-generation of the Cytoscape Exchange format designed to address limitations of its predecessor. CX2 introduces a simplified data model, optimized for memory-efficient processing of large-scale networks via enhanced streaming capabilities, and a more compact serialization to improve data transfer rates and reduce storage requirements. RCX2 provides a comprehensive programmatic interface within R for creating, modifying, serializing, and deserializing biological networks conforming to the CX2 standard. A critical design feature of the RCX and RCX2 packages is their inherent interoperability. The underlying data models facilitate direct, lossless conversion between each other, and as a consequence between the CX and CX2 formats. This deliberate compatibility ensures a smooth transition for users adopting the newer standard while maintaining the ability to interact with existing CX-formatted data and collaborate with researchers using the original specification. Although RCX and RCX2 employ distinct internal data handling strategies tailored to the nuances of each format, their synergistic relationship provides a unified and forward-compatible solution for the R-based biological network research community. Together, these packages offer a robust framework for managing biological network data across different versions of the Cytoscape Exchange format, future-proofing analytical pipelines and fostering seamless data exchange within the broader network biology landscape.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Lucas Beerland
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Interpretable Distributional Inference in Single-Cell Proteomics" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-25-contents" aria-controls="callout-25" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Interpretable Distributional Inference in Single-Cell Proteomics
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-25" class="callout-25-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Recent advancements in mass spectometry based single cell proteomics (SCP) enabled the characterization of cellullar heterogeneity across conditions at unprecedented resolution. However, current SCP data analysis workflows still focus on comparing average protein abundances and overlook informative distributional changes in shape, such as differences in variability and/or modality, limiting the advantage of SCP over bulk proteomics. We therefore propose a novel statistical framework for SCP data to infer distributional differences between conditions. Our method builds on Lindsey’s Method, which recasts the density estimation into a Poisson regression problem i.e., by fitting smooth histograms with a large number of equally spaced bins using a basis function expansion. In this contribution we enable interpretable inference on differences in the abundance distributions between conditions by using interactions between the spline basis and an experimental factor. We illustrate how our modelling approach can prioritize proteins that exhibit differential distributions across conditions using likelihood ratio tests and Wald-tests. These tests assess the omnibus null hypothesis of a common density across conditions. Next, we develop tests on pairwise contrasts between the groupwise smoothers to infer regions with distributional differences between conditions. This approach also enabled us to provide our users with intuitive plots that visualize the density estimators in both conditions, highlight regions with differential distributions, and have a one-to-one relation to the models and hypothesis tests that we propose. Finally, we evaluate and illustrate our novel framework using simulation studies and real SCP experiments. These analyses show that our comprehensive distributional analysis framework is a first step to leverage the wealth of information in SCP data. It offers a novel perspective to study single-cell heterogeneity and to compare the protein abundance distribution in populations of single cells that differ in cell type, biological conditions, or treatment.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; background-color: lightyellow !important;padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
12:30
</td>
<td style="text-align:left;width: 20%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
Lunch
</td>
<td style="text-align:left;width: 25%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; background-color: lavender !important;padding-left: 2em;border-bottom: 1px solid lightyellow" indentlevel="1">
13:45
</td>
<td style="text-align:left;width: 20%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Keynote
</td>
<td style="text-align:left;width: 25%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Susan Holmes
</td>
<td style="text-align:left;width: 40%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Latent variables as the best medicine for heterogeneity
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
14:15
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Poster pitches
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
14:45
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Short talks
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Jacopo Ronchi
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="MIRit: an integrative R framework for the identification of impaired miRNA-mRNA regulatory networks in complex diseases" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-24-contents" aria-controls="callout-24" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
MIRit: an integrative R framework for the identification of impaired miRNA-mRNA regulatory networks in complex diseases
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-24" class="callout-24-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>MicroRNAs (miRNAs) are critical regulators of gene expression, implicated in nearly all cellular processes and frequently associated with pathological development. Given their broad functional significance, systematic and accurate characterization of miRNA dysregulation is essential to uncover the molecular underpinnings of disease. While high-throughput technologies such as microarrays and miRNA-Seq enable the quantification of small RNA transcripts, deciphering their functional roles in disease mechanisms remains a major challenge. This is partly due to the lack of standardized, integrative frameworks for miRNA-mRNA analysis and the widespread reliance on outdated or inappropriate methodologies, which often yield poorly reproducible results and limited insights into miRNA regulatory networks. Moreover, the absence of statistical models tailored for non-sample-matched datasets significantly hampers the exploitation of many existing miRNA datasets, further restricting their biological interpretability. To address these limitations, we present MIRit, an open-source, comprehensive R framework designed to support rigorous, state-of-the-art integrative analyses of miRNA and mRNA data. MIRit guides users through all critical steps, including differential expression analysis of miRNAs and mRNAs, and identification of miRNA-target pairs using ensemble-based prediction methods combined with validated interactions. It further integrates miRNA and mRNA expression profiles using statistically robust techniques such as partial correlation analysis, rotation gene set tests, and one-sided association tests. Beyond integration, MIRit offers a suite of tools for exploring dysregulated miRNA networks, including the identification of disease-associated variants affecting miRNA expression and the assessment of the functional impact of miRNA dysregulation.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Najla Abassi
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="One class to rule it all: DeeDeeExperiment for managing and exploring omics analysis results" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-23-contents" aria-controls="callout-23" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
One class to rule it all: DeeDeeExperiment for managing and exploring omics analysis results
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-23" class="callout-23-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Differential expression analysis (DEA) and functional enrichment analysis (FEA) are core steps in transcriptomic workflows, enabling researchers to detect and interpret biological differences between conditions. However, managing the outputs of these analyses across multiple contrasts becomes increasingly overwhelming, especially when the number of conditions (defined by complex experimental designs) and analysis tools increases. This challenge is further amplified in single-cell RNA-seq, where pseudo-bulk analyses generate numerous results tables across cell types and contrasts. As a result, efficient management, exploration, interpretation and reproducibility of these outputs becomes a significant challenge, also for experienced practitioners. To address these issues, we present DeeDeeExperiment, a new R/S4 class that extends the widely adopted SummarizedExperiment core Bioconductor object. DeeDeeExperiment provides a structured and consistent framework to organize DEA and FEA results alongside the core expression data and metadata, enabling users to retrieve and explore analysis outputs across multiple contrasts in a coherent and easy manner. DeeDeeExperiment is fully compatible with the Bioconductor ecosystem, promoting reproducibility and enabling seamless integration not only with existing popular visualization tools, such as GeneTonic and iSEE, but also making it easy to be plugged into existing workflows that are based on “classical” SummarizedExperiment objects and its derivatives. DeeDeeExperiment is publicly available under the MIT license at https://github.com/imbeimainz/DeeDeeExperiment</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Justine Leclerc
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="conformeR: Conformalized differential expression analysis of multi-condition single-cell data" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-22-contents" aria-controls="callout-22" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
conformeR: Conformalized differential expression analysis of multi-condition single-cell data
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-22" class="callout-22-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Multi-condition single-cell RNA sequencing reveals how gene expression varies across conditions within specific cell populations. Most current methods model these changes by fitting gene-wise generalized linear models to read counts, and then detect differential expression using statistical tests such as the likelihood-ratio test or the quasi-likelihood F-test . Each gene <span class="math inline">\(\times\)</span> cell pair occurs only once in the dataset and is observed under a single condition. Predicting counterfactual expression levels—e.g., estimating how a cell would express genes under treatment even if observed under control—increases sample size by imputing expression values for all conditions. This might reduce false discovery rates and improve detection power, both key to efficient statistical testing. The method LEMUR addresses this imputation task using a type of PCA that learns for each cell a low-rank structure per condition. The flexibility of the model makes uncertainty more challenging to quantify, an open problem as stated by the authors of LEMUR. Our conformeR is a wrapper around LEMUR R-package that adds uncertainty quantification without altering the underlying model. conformeR constructs prediction intervals for LEMUR-imputed values with finite-sample coverage guarantees using conformal prediction , which relies on the assumption that cells from the same biological replicate and cell type are exchangeable. By inverting conformal prediction intervals, conformeR outputs for each gene <span class="math inline">\(\times\)</span> cell pair a p-value that encodes the difference between the observed and predicted expression levels. We then adjust these p-values using the Benjamini-Hochberg procedure , leveraging the fact that conformal p-values satisfy the positive regression dependence on a subset condition . Finally, conformeR aggregates over cells from the same replicate and cell type to yield a single p-value per gene. We evaluate conformeR on scRNA-seq data from 864 patients with varying Covid-19 severity . The large cohort allows us to account for patient-level variability and demonstrate the robustness and power of conformeR. Bibliography (bib format) <span class="citation" data-cites="article">@article</span>{lemur, author = {Ahlmann-Eltze, Constantin AND Huber, Wolfgang }, title = {Analysis of Multi-Condition Single-Cell Data with Latent Embedding Multivariate Regression}, journal = {Nature Genetics}, volume = “57”, pages = “659–667”, year = “2025” } <span class="citation" data-cites="article">@article</span>{pval_prds, author = {Bates, Stephen AND Candès, Emmanuel AND Lei, Lihua AND Romano, Yaniv AND Sesia, Matteo }, title = {Testing for Outliers with Conformal P-Values}, journal = “The Annals of Statistics” , volume = “51”, pages = “149–178”, year = “2023” } <span class="citation" data-cites="article">@article</span>{bh_correction, author = {Benjamini, Yoav AND Hochberg, Yosef }, title = {Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing}, journal = “Journal of the Royal Statistical Society: Series B (Methodological)”, volume = “57”, number = “1”, pages = “289–300”, year = “1995” } <span class="citation" data-cites="article">@article</span>{scMerge2, author = “Lin, Yingxin AND Cao, Yue AND Willie, Elijah AND Patrick, Ellis AND Yang, Jean Y. H.”, title = “Atlas-scale single-cell multi-sample multi-condition data integration using {scMerge2}”, journal = “Nat Commun”, volume = “14”, number = “4272”, DOI = “10.1038/s41467-023-39923-2”, year = “2023” } <span class="citation" data-cites="article">@article</span>{edgeR, author = “Robinson, Mark D. AND McCarthy, Davis J. AND Smyth, Gordon K.”, title = “{edgeR}: a Bioconductor package for differential expression analysis of digital gene expression data”, journal = “Bioinformatics”, volume = “26”, pages = “139-40”, DOI = “10.1093/bioinformatics/btp616”, year = “2010” } <span class="citation" data-cites="book">@book</span>{cpvovk, author = “Vovk, Vladimir AND Gammerman, Alex AND Shafer, Glenn”, title = “Algorithmic Learning in a Random World”, publisher = “Springer”, year = “2005” }</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Lizhong Chen
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="edgeR v4 with expanded functionality and improved support for small counts and larger datasets" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-21-contents" aria-controls="callout-21" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
edgeR v4 with expanded functionality and improved support for small counts and larger datasets
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-21" class="callout-21-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>edgeR is an R/Bioconductor software package for differential analyses of sequencing data in the form of read counts for genes or genomic features. Over the past 15 years, edgeR has been a popular choice for statistical analysis of data from sequencing technologies such as RNA-seq or ChIP-seq. This year, we announce edgeR version 4 with expanded functionality and improved support for small counts and larger datasets. We introduce a new quasi-likelihood (QL) method in edgeR 4.0.0, using the adjusted deviance statistics to yield very nearly unbiased quasi-dispersion estimators even for fitted values close to zero. We only estimate a constant NB-dispersion from the most highly expressed genes which is fast even for larger dataset. We update the empirical Bayes hyperparameter estimation in edgeR 4.4.0, to give improved performance when the residual degree of freedom are unequal and possibly small. Recently we introduce a new diffSplice method for alternative splicing analysis in edgeR 4.6.0. With those expanded functionality, we can apply edgeR to single-cell RNA-seq data analysis. For example, we can identify the highly variable genes using the goodness-of-fit test for one sample or multiple samples. We can also find the marker genes for the cell clusters using one vs the average of others approach instead of pseudo-bulk approach which only works for more than two samples. Besides, the new diffSplice method is designed for alternative splicing by differential transcript usage or differential exon usage analysis. We are continuing to work on the edgeR project to expand the functionality and introduce new statistical methods, such as testing relative to a fold-change threshold, introducing the sample weights to account for the variations in sample quality and so on in the future.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Koen Van den Berge
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Transcription factor activity estimation via probabilistic gene expression deconvolution" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-20-contents" aria-controls="callout-20" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Transcription factor activity estimation via probabilistic gene expression deconvolution
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-20" class="callout-20-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Gene expression is the primary modality being studied to differentiate between biological cells. Contemporary single-cell studies simultaneously measure genome-wide transcription levels for thousands of individual cells in a single experiment. While the characterization of cell population differences has often occurred through differential expression analysis, tiny effect sizes become statistically significant when thousands of cells are available for each population, compromising biological interpretation. Moreover, these large studies have spurred the development of methods to infer gene regulatory networks (GRNs) directly from the data, and GRN databases are becoming more comprehensive. In this work, we propose a statistical model for gene expression measures and an inference method that leverage GRNs to deconvolve transcription factor (TF) activity from gene expression, by probabilistically assigning mRNA molecules to TFs. This shifts the paradigm from investigating gene expression differences to regulatory differences at the level of TF activity, aiding interpretation and allowing prioritization of a limited number of TFs responsible for significant contributions to the observed gene expression differences. The inferred TF activities result in intuitive prioritization of TFs in terms of the (difference in) estimated number of molecules they produce, in contrast to other widely-used methods relying on arbitrary enrichment scores. Our model allows the incorporation of prior information on the regulatory potential between each TF and target gene through prior distributions, and is able to deal with both repressing and activating interactions. We compare our approach to other TF activity estimation methods using simulation experiments and case studies.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Nicolò Gnoato
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="A Computational Approach to Characterize Tumour Cells via Integrated CNV Metrics from scRNAseq Data" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-19-contents" aria-controls="callout-19" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
A Computational Approach to Characterize Tumour Cells via Integrated CNV Metrics from scRNAseq Data
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-19" class="callout-19-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Cancer is a complex pathological condition that originates from the accumulation of genetic mutations, which can manifest as both point mutations in single nucleotides and structural modifications of the genome, such as copy number variations (CNVs). Evidence has demonstrated that CNVs significantly alter gene expression levels, disrupting normal cellular mechanisms and promoting uncontrolled cell proliferation. Tumour formation, resulting from this abnormal growth, ultimately damages surrounding tissues and impairs physiological functions. Therefore, characterising CNVs is crucial for investigating the mechanisms underlying tumorigenesis and tumour progression. Currently, cell classification largely relies on marker genes, an approach limited by marker selection biases and methodological efficiency, particularly in cancer cell detection. To overcome these limitations, we are developing a R package aimed at effectively stratifying normal and tumour cells using their CNV profiles inferred from single-cell RNA sequencing data (scRNAseq). Our approach integrates several metrics, including copy number burden (CNB), ploidy, CNV signature features, and homologous recombination deficiency (HRD), to robustly identify tumour cells. Given the known instability of cancer genomes, our strategy calculates CNV-based scores across the entire transcriptome and within specific genomic regions known to be associated with distinct tumour subtypes. This scoring enhances classification accuracy, providing greater confidence in distinguishing tumour from non-tumour cells. Ultimately, our methodology offers a refined approach to cancer cell identification and characterization through comprehensive CNV analysis, potentially advancing our understanding of tumour biology and informing therapeutic strategies.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Anna C E De Lima Tanada
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Integration of multi-omics data with topological pathway analysis tool: case studies with MOSClip" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-18-contents" aria-controls="callout-18" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Integration of multi-omics data with topological pathway analysis tool: case studies with MOSClip
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-18" class="callout-18-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Singular and individual gene approaches became obsolete with the emergence of omics technologies. For example, omics-focused studies of biological processes can paint a more holistic picture of complex diseases, such as cancer or neurodegenerative diseases. From this perspective, we developed our R package MOSClip, which was recently added to BioConductor. MOSClip integrates multiple omics by implementing graph theory methods for topological analysis of biological pathways. It performs dimensionality reduction, which can be done at two levels: whole pathways or decomposition of pathways into modules. Then, MOSClip can either evaluate the impact of each pathway and omic on patients’ prognosis through survival analysis, or operate a two-class analysis. Additionally, MOSClip offers multiple graphical tools to aid in the visualization of the results. The last released version of our R package supports bulk RNA-sequencing, methylation, mutation, and copy number variation data. A new addition to MOSClip is the support for ATAC-seq data, and we are further developing it to accept single-cell data. To illustrate and confirm the utilities and performance of this package, we conducted multiple case studies using multi-omics data from different complex diseases. Specifically, we focused on the new functionalities of MOSClip, including the two-class analysis. MOSClip is a valuable tool for its ability to give powerful insights into complex diseases that could pass undetected otherwise. This is achieved thanks to the topological analysis and the built-in MOSClip graph functions, allowing quick and intuitive interpretation of the results. In summary, we showed that MOSClip can extract powerful information from complex and intricate data from omics integration, demonstrating its importance for researching several diseases.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Igor Cervenka
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Rega - R Interface for Metadata Submission to European Genome-Phenome Archive" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-17-contents" aria-controls="callout-17" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Rega - R Interface for Metadata Submission to European Genome-Phenome Archive
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-17" class="callout-17-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The European Genome-phenome Archive (EGA) is offers secure, long-term storage and controlled access to personally identifiable genetic and phenotypic datasets. While the EGA’s website enables data submission, manual entry is error-prone and time-consuming for larger datasets. Crafting and validating the complex metadata payloads needed for dataset deposition remains a persistent bottleneck for many laboratories. We present Rega, an open-source R package that leverages API provided by EGA, enabling programmatic interaction. Rega simplifies and systematizes metadata submission by coupling an intuitive, GEO-style Excel template with a robust, extensible R interface that: Transforms—converts user-filled spreadsheets or in-memory R data frames into EGA-compliant JSON payloads. Validates—performs schema-aware checks on samples, experiments, datasets and analyses, flagging structural or semantic errors before transmission. Uploads—leverages the EGA REST API with built-in retries and granular progress reporting, delivering reduction in submission time compared to manual web-form entry. By abstracting away low-level API intricacies and unifying metadata management in a familiar spreadsheet-to-R paradigm, Rega lowers the barrier to data sharing, enhances reproducibility and encourages timely deposition of sensitive genomic resources. Rega is released under the Artistic license and is hosted on GitHub, with comprehensive vignettes.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
16:30
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Poster session
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
<tr grouplength="25">
<td colspan="4" style="border-bottom: 1px solid #eee; ">
<strong>Thu. - Sep.&nbsp;18, ’25</strong>
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; background-color: lavender !important;padding-left: 2em;border-bottom: 1px solid lightyellow" indentlevel="1">
9:00
</td>
<td style="text-align:left;width: 20%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Keynote
</td>
<td style="text-align:left;width: 25%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
James Sharpe
</td>
<td style="text-align:left;width: 40%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
C3PO: Cell 3D Positioning by Optical encoding and its application to spatial transcriptomics
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
9:30
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Short talks
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Charlotte Soneson
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Efficient representation and analysis of single molecule footprinting data with footprintR" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-16-contents" aria-controls="callout-16" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Efficient representation and analysis of single molecule footprinting data with footprintR
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-16" class="callout-16-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Single molecule footprinting is an increasingly used assay to study gene regulation and chromatin biology using enzymatic modification of accessible DNA, followed by detection of modifications by sequencing. The resulting data provides genome-wide, rich measurements of accessibility and DNA modifications at near-base pair resolution and relatively low cost. However, the representation of such data is not yet standardised and only few tools have been specifically developed to handle it. Here we present footprintR, a new R package that provides a framework for representing and analysing single molecule footprinting data. Read-level and summarized data can be imported from standard file types and are stored together in a single SummarizedExperiment container, using the newly developed NaMatrix from the SparseArray Bioconductor package for efficient representation and computation. We apply footprintR to genome-wide 6mA and 5mC footprinting data obtained using nanopore long read sequencing and illustrate how it can be used to address various biological questions, including footprint scoring, nucleosome placement and measurement of nucleosome spacing, as well as detection of differentially methylated or accessible regions. footprintR also contains flexible and powerful visualisation functionalities. Single molecule footprinting is becoming a more and more important part of the epigenetics and gene regulation research toolbox. We hope that footprintR with its efficient data representation and analysis functions will facilitate the analysis of such data for R users.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Ning Shen
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="decemedip: hierarchical Bayesian modeling for cell type deconvolution of immunoprecipitation-based DNA methylomes" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-15-contents" aria-controls="callout-15" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
decemedip: hierarchical Bayesian modeling for cell type deconvolution of immunoprecipitation-based DNA methylomes
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-15" class="callout-15-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>MeDIP-seq is an enrichment-based DNA methylation profiling technique that measures the abundance of methylated DNA. While this technique offers efficiency advantages over direct methylation profiling, it does not provide absolute quantification of DNA methylation necessary for cell type deconvolution. We introduce decemedip, a Bayesian hierarchical model for cell type deconvolution of methylated sequencing data that leverages reference atlases derived from direct methylation profiling. We demonstrate its accuracy and robustness through simulation studies and validation on cross-platform measurements, and highlight its utility in identifying tissue-specific and cancer-associated methylation signatures using MeDIP-seq profiling of patient-derived xenografts and cell-free DNA. decemedip is available at .</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Jiayi Wang
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Bias Correction and Differential Motif Accessibility in ATAC-seq Data" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-14-contents" aria-controls="callout-14" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Bias Correction and Differential Motif Accessibility in ATAC-seq Data
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-14" class="callout-14-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The Assay for Transposase-Accessible Chromatin using sequencing (ATAC-seq) has become widely adopted for assessing chromatin accessibility due to its speed, simplicity, and low input requirements. These advantages make ATAC-seq particularly well-suited for identifying transcription factors (TFs) that mediate regulatory changes between cell types or states. However, technical biases inherent to ATAC-seq data can complicate downstream analyses, especially the accurate identification of differentially active TFs. A recent benchmarking study identified a variation of <span class="math inline">\(chromVAR\)</span> as the preferred method for this task, but we observed that it suffers from an incomplete and computationally intensive bias correction strategy and overlooks the footprint characteristics of TF binding. To address these limitations, we developed two alternative approaches: the weighted and insertion models. The weighted model corrects GC-content and fragment length biases in ATAC-seq data by assigning weights to fragments, followed by cyclic loess normalization to calculate peak weights for enrichment bias correction. The resulting unbiased peak accessibility matrix is then used for differential motif accessibility analysis with <span class="math inline">\(limma\)</span>-<span class="math inline">\(voom\)</span>. Unlike <span class="math inline">\(chromVAR\)</span>, which generates motif-level scores, the unbiased peak-level accessibility count matrix produced by the weighted model enables broader downstream analyses, such as peak-level differential accessibility analysis and bias-corrected coverage profiles for visualization. We therefore envision that the weighted model will have broader applicability to bulk epigenetic data, providing unbiased information for a variety of downstream analyses at multiple levels. The insertion model enhances differential motif accessibility analysis by incorporating Tn5 footprint patterns around motif sites, assigning weighted insertion scores per motif match. Bias correction is performed using <span class="math inline">\(chromVAR\)</span>-like background peaks, and differential motif accessibility is assessed using z-scores and <span class="math inline">\(limma\)</span>. Each of these two models offers enhanced statistical performance, with the weighted model effectively correcting various technical biases. As a future direction, we aim to integrate the weighted and insertion models into a unified framework that leverages both fragment-level bias correction and motif-specific footprint signals for more robust differential motif accessibility analysis.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; background-color: lightyellow !important;padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
10:15
</td>
<td style="text-align:left;width: 20%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
Break
</td>
<td style="text-align:left;width: 25%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
11:00
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Short talks
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Maria Doyle
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Empowering Bioinformatics in Africa through Bioconductor: Expanding Training and Community Engagement" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-13-contents" aria-controls="callout-13" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Empowering Bioinformatics in Africa through Bioconductor: Expanding Training and Community Engagement
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-13" class="callout-13-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>For over 20 years, Bioconductor (https://bioconductor.org/) has provided an open-source ecosystem for reproducible genomic data analysis, supporting over 2,000 R-based packages with 1M+ annual unique downloads worldwide. However, access to bioinformatics training remains a challenge in Africa, limiting the adoption of these tools. To bridge this gap, Bioconductor is expanding its training initiatives and community collaborations across the continent. Through the Bioconductor Carpentry program, we have trained 31 instructors in the last two years, developed structured training materials on R/Bioconductor, RNA-seq and single-cell analysis, and delivered 28+ workshops globally. To address the growing demand for in-person training, we have partnered with local bioinformatics leaders to deliver on-site workshops in East and West Africa. In March 2025, we held a highly successful week-long workshop in Nairobi, Kenya, in collaboration with the International Institute of Tropical Agriculture (IITA) and international experts. The course provided hands-on training in genomic data analysis and RNA-seq workflows, enabling participants to apply Bioconductor tools to their own datasets. More details: training.bioconductor.org/workshops/2025-03-Nairobi . We plan to run a similar course in West Africa in late 2025 / early 2026. We aim to connect with and support the African bioinformatics community by: - Expanding instructor networks to increase local training capacity. - Identifying needs and gaps in bioinformatics training and tools to ensure our initiatives are needs-based. - Developing region-specific curricula for One Health research, including pathogen and agricultural genomics. - Collaborating with African bioinformatics institutions to scale training opportunities. - Providing opportunities to get involved in developing training materials in local languages. This presentation will highlight progress to date, lessons learned, and future opportunities, inviting researchers and educators from around the world to join Bioconductor’s growing global community and contribute to bioinformatics education.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Luca Marconato
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Advancing spatial biology analysis across programming languages" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-12-contents" aria-controls="callout-12" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Advancing spatial biology analysis across programming languages
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-12" class="callout-12-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The rapid growth and increasing diversity of spatial profiling technologies, combined with the demand for accelerated innovation cycles, have created a fragmented landscape characterized by incompatible file formats and isolated analysis methods. This fragmentation significantly hinders scientific reproducibility. To address these challenges, scverse, in collaboration with napari and researchers from the Open Microscopy Environment (OME), developed the SpatialData framework. This framework provides a language-agnostic, standardized storage format derived from the OME-Zarr format and OME-NGFF specification, specifically designed to maximize interoperability for spatial omics. Additionally, the SpatialData framework includes a suite of Python libraries tailored for high-performance processing and flexible visualization of spatial omics data. SpatialData aims to ease interoperability of analysis and visualization tools via its standardized storage format. In this talk, I will share recent developments of the SpatialData framework, emphasizing efforts aimed at extending interoperability beyond Python. I will present preliminary results from ongoing collaborations with the Bioconductor, and bioimaging (napari, OME) communities, highlighting key challenges, implemented solutions, and our vision for fully interoperable, cross-language analyses of spatial omics data. These efforts aim to leverage the unique strengths of each programming language and their respective analysis communities. Furthermore, I will stress the critical importance of adopting open, reusable standards to mitigate redundancy and fragmentation. Specifically, I will discuss ongoing efforts within the bioimaging community to develop a canonical parser for OME-NGFF and OME-Zarr formats in Python, which will significantly enhance interoperability and facilitate the development of sustainable, maintainable software solutions. Similar efforts within Bioconductor have the potential to significantly broaden accessibility, foster community collaboration, and unlock powerful, integrative spatial omics analyses across diverse programming environments.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Peiying Cai
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Detection of differential spatial patterns in spatial omics data" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-11-contents" aria-controls="callout-11" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Detection of differential spatial patterns in spatial omics data
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-11" class="callout-11-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Background Spatially resolved transcriptomics (SRT) technologies measure gene expression while preserving spatial context. Several methods have been developed to identify spatially variable genes (SVGs), i.e., genes whose expression profiles vary across tissue. As multi-sample, multi-condition SRT datasets (e.g., healthy vs.&nbsp;diseased, different treatments, or time points) become more common, it can be of interest to identify differences between conditions, such as genes with different abundances or splicing patterns. However, current methods face several limitations: (i) lack of support for multiple samples and conditions; (ii) high computational demands; and, (iii) inability to test for spatial changes of gene expression patterns across groups. Methodology Here, we present an extension of our previous DESpace framework for identifying genes with differential spatial patterns (DSP) across conditions in multi-sample settings. The method inputs pre-computed spatial domains, i.e., regions of neighboring spots with similar expression profiles, usually identified by spatially resolved clustering tools. We then pseudo-bulk gene expression within each domain and sample, and use edgeR with spatial clusters and conditions as covariates. Intuitively, if the expression is significantly associated with the interaction terms between spatial clusters and condition, the spatial gene expression structure varies between conditions, hence indicating DSP. Key strengths of our method include: (i) modeling of multiple samples across multiple conditions, reducing uncertainty from individual samples and identifying genes with DSP across experimental conditions; (ii) region-specific testing, allowing investigation of the mRNA abundance changes between conditions in areas of particular interest; and, (iii) computational efficiency and compatibility with diverse SRT platforms. Benchmarking We benchmarked our method against scran’s findMarkers, Seurat’s FindMarkers (with and without pseudo-bulk aggregation), and spatialLIBD, using semi-simulated datasets generated from two real SRT datasets, with four known spatial expression patterns. In simulations, our approach exhibits well-calibrated false discovery rates and higher true positive rates, while on real datasets, our framework identified more condition-associated genes than the competitors. Availability Our approach is available via Bioconductor: https://bioconductor.org/packages/DESpace. A pre-print (in preparation) will also follow in the coming weeks.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Martin Emons
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Differential spatial co-localisation analysis of multi-sample and multi-condition experiments with spatialFDA" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-10-contents" aria-controls="callout-10" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Differential spatial co-localisation analysis of multi-sample and multi-condition experiments with spatialFDA
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-10" class="callout-10-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Spatial omics technologies are generating vast amounts of spatial data with often complex experimental designs and the need for tested tools to quantify spatial features from these datasets is becoming increasingly important. There are many standard tools to quantify different aspects of spatial data (e.g.&nbsp;cellular colocalisation). Most of these tools do not provide a way to compare multiple samples across conditions with statistical models. The existing approaches that allow for such a comparison across samples focus on comparing scalar metrics. Our Bioconductor package, spatialFDA, takes a different approach by using spatial metrics that quantify so-called <span class="math inline">\(r\)</span>-neighbourhoods with increasing radii <span class="math inline">\(r\)</span>, thereby constructing the spatial metric as function of the radius <span class="math inline">\(r\)</span>. Instead of compressing these functions into scalar values, spatialFDA compares the entire functions between conditions using generalised additive mixed models with functional responses. This estimation framework allows to account for the complex correlational structure of spatial omics datasets. The estimated coefficients of these models are themselves functions over the user-defined radius range. This allows for an interpretation of the scale of the differential co-localisation effect between conditions. The methodology of spatialFDA was extended and improved to reflect the characteristics of spatial statistics functions. This entailed improved estimation of random errors, optimising the identifiability of the model and specifying suitable link functions. We show that spatialFDA can be used to find biologically-meaningful differential co-localisation of cell types in different biological examples. We have developed simulation scenarios to investigate the performance of spatialFDA in a controlled setting and compare it to competing methods.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Ilaria Billato
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="A standardized R/Bioconductor framework for integrative analysis of histopathological images with multi-omics data" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-9-contents" aria-controls="callout-9" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
A standardized R/Bioconductor framework for integrative analysis of histopathological images with multi-omics data
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-9" class="callout-9-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Histopathological images provide unparalleled insights into tissue architecture, cellular morphology, and tumor spatial organization. While these images are routinely used in cancer research and diagnosis, their computational analysis typically requires specialized software outside the R/Bioconductor ecosystem, the primary environment for other high-throughput omics analyses. This disconnection creates a significant barrier to integrative multi-model analyses. To address this challenge, we aim to develop standardized workflows for processing raw histopathological images and making extract features available in R, along with a comprehensive repository of pre-processed features from The Cancer Genome Atlas (TCGA) images. We developed a comprehensive image analysis pipeline that includes color normalization, tissue segmentation, tiling, and feature extraction at the nuclei and region-of-interest (ROI) levels. The pipeline leverages state-of-the-art deep learning models, including HoVer-Net for nuclear segmentation and classification, and outputs standardized data structures compatible with R/Bioconductor’s existing analytical frameworks. We validated this workflow by processing the complete collection of diagnostic H&amp;E-stained whole slide images (WSIs) from TCGA. We established a standardized histopathology image analysis pipeline and implemented it on 11,765 diagnostic images from 9,640 cases across 25 cancer types. Extracted nuclei-level features included location, shape, texture, and classification (e.g., benign, neoplastic, stromal and necrotic), while ROI-level features captured tissue organization, cellular composition, and spatial relationships. We further demonstrated the use case of multi-model integration across extracted image features, genomics, and transcriptomics data. Finally, to enhance the usability of this resource, we developed ImageTCGA, a Shiny application for interactive exploration, filtering, and visualization of the extracted features with the original images. Our standardized workflow and feature repository bridge a critical gap between histopathological image analysis and multi-omics integration. By providing robust tools within the R/Bioconductor environment, along with pre-computed features from TCGA, we enable researchers to readily incorporate spatial and morphological insights into their analyses. ImageTCGA facilitates hypothesis generation and validation across diverse cancer types, potentially accelerating biomarker discovery and advancing our understanding of tumor-microenvironment interactions. Future work will focus on expanding the repository to include additional cancer image collections and developing new analytical methods for multi-modal data integration.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Maryna Chepeleva
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Functional Signal Decomposition in Spatial Transcriptomic and Multi-Omic Data" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-8-contents" aria-controls="callout-8" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Functional Signal Decomposition in Spatial Transcriptomic and Multi-Omic Data
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-8" class="callout-8-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Interpreting spatial multi-omics data in terms of biologically meaningful functional programs remains a key challenge in computational biology. We present recent developments and use cases for extracting such signals using our R package, consICA, which includes signal deconvolution method followed by functional annotation. Originally developed for robust extraction of mixed signals from multi-omics data, consICA has now been extended to support spatial analysis. We applied consICA to the whole tissue slide in a spatial transcriptomics dataset. The resulting spatially resolved independent components revealed biologically interpretable gradients of transcriptional activity, including angiogenesis, immune response, and cell death programs. In parallel, we used ssGSEA on the same spatial matrices, treating spatial spots as samples. This enables local estimation of pathway activities at single-spot resolution, providing 2D tissue maps of metabolic and regulatory programs. In addition to spatial data, we demonstrate applications to multi-omics cancer datasets, including metabolome-proteome data from glioblastoma. This analysis uncovers coordinated functional signals across data modalities and supports the generation of interpretable biological hypotheses.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Liyang Fei
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="barbieQ: An R software package for analysing barcode count data from clonal tracking experiments" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-7-contents" aria-controls="callout-7" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
barbieQ: An R software package for analysing barcode count data from clonal tracking experiments
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-7" class="callout-7-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>In cancer and stem cell research, comparing the clonal composition of pools of cells before and after perturbations can reveal important features such as treatment-resistance or multipotent clones. New clonal tracking technologies now allow us to track progeny cells back to their original progenitor cell using unique DNA barcodes that are passed from each original cell to its offspring. The barcode count for each sample indicates the number of progeny cells derived from the original cell. However, there is a lack of bioinformatic tools for robust statistical analysis of barcode count data from bulk sequencing of DNA barcodes. To address this, we introduce barbieQ, an Bioconductor R package designed for analysing barcode count data across groups of samples. barbieQ is built upon the widely used SummarizedExperiment data structure, making it interoperable with low-level processing tools. It provides novel functions for data cleaning, summarising, and visualising barcode count data, and implements two statistical tests: 1) Differential barcode proportion - identifies barcodes with significantly changing proportions in one condition compared to another. 2) Differential barcode occurrence - identifies barcodes more frequently seen in the samples of one condition compared to another. barbieQ can handle complex experimental designs using regression models to test a factor of interest while taking into account other variables. Both tests maintain a type 1 error less than 5% under the null hypothesis. Notably, in analysis of a real world dataset of monkey blood stem cell differentiation, barbieQ accurately identified novel NK cell-specific clones whilst avoiding less specific barcodes, in contrast to the original analysis based on heuristic cutoffs and visualisations commonly used in the field. Overall, barbieQ is a powerful Bioconductor package with greater power to detect biologically meaningful changes compared to traditional methods. The package is available for download in the development version of Bioconductor using BiocManager::install(“barbieQ”).</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; background-color: lightyellow !important;padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
12:30
</td>
<td style="text-align:left;width: 20%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
Lunch
</td>
<td style="text-align:left;width: 25%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; background-color: lavender !important;padding-left: 2em;border-bottom: 1px solid lightyellow" indentlevel="1">
13:45
</td>
<td style="text-align:left;width: 20%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Keynote
</td>
<td style="text-align:left;width: 25%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Noelia Ferruz
</td>
<td style="text-align:left;width: 40%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Controllable protein design with language models
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
14:15
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Poster pitches
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
14:45
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Workshops
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Yixing Dong
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="OSTA: Orchestrating Spatial (Transcript-)Omics Analysis with Bioconductor" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-6-contents" aria-controls="callout-6" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
OSTA: Orchestrating Spatial (Transcript-)Omics Analysis with Bioconductor
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-6" class="callout-6-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Spatial omics technologies have advanced rapidly over the past five years, spanning both sequencing-based and imaging-based platforms, with applications across diverse tissues such as the brain and cancer. However, despite these technological leaps, many current analysis pipelines have been adapted from single-cell RNA-seq workflows without sufficient tailoring to the unique spatial context. As a result, a comprehensive, statistically-informed framework for spatial transcriptomics analysis remains largely absent. To address this gap, we have collaboratively developed an open-source online book that serves as the most comprehensive Bioconductor-based resource for spatial transcriptomics to date (https://lmweber.org/OSTA/). This book offers step-by-step tutorials built around publicly available datasets, covering a broad spectrum of spatial technologies and resolutions, from Visium, Visium HD, to Xenium and CosMx. We showcase workflows that incorporate spatial statistical methods for pattern detection, modeling spatial variability, and integrating multimodal data. Our resource also emphasizes reproducibility and interoperability between R and Python data classes and tools. In this workshop, we will present newly developed workflow chapters that establish various avenues of spatial omics data analysis. We will also demonstrate complete pipelines—from raw data ingestion to spatial visualization and biological interpretation—designed to be both user-friendly and extensible to emerging technologies. By promoting open standards and cross-language compatibility, our work aims to equip the spatial omics community with reproducible, robust, and accessible analytical tools.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Ellis Patrick
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Making Space Count: Strategies for Analysing Spatial Omics Data" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-5-contents" aria-controls="callout-5" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Making Space Count: Strategies for Analysing Spatial Omics Data
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-5" class="callout-5-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Recent advances in highly multiplexed cell imaging technologies—such as PhenoCycler, IMC and CosMx, Xenium—have fundamentally transformed our ability to study complex cellular relationships within tissues. While traditional immunohistochemistry protocols were limited to visualising cells based on just two or three surface proteins, these cutting-edge technologies enable profiling of dozens of proteins or thousands of RNA molecules in situ. This breakthrough enables precise classification of cell subtypes and offers an unprecedented view of cellular heterogeneity in tissue environments. Extracting biological meaning from spatial omics data requires not just new methods, but a solid appreciation for when and where applying them will provide insight. Here we will introduce new functionality in our Bioconductor packages spicyR, Statial, and lisaClust, designed to help quantify how changes in cell interactions, spatial heterogeneity, and microenvironment composition are associated with disease. We frame this new functionality within our spatial playbook—a guide to selecting the right analytical strategy for different research questions and data types. Just as no single tactic works for every opponent in sport, no single method suits every spatial omics dataset. Our goal is to provide analysts with a practical guide for making informed, strategic choices in spatial analysis—making complex spatial omics data easier to interpret and more suitable for generating robust, reproducible conclusions.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Tuomas Borman
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Orchestrating Microbiome Analysis with Bioconductor" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-4-contents" aria-controls="callout-4" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Orchestrating Microbiome Analysis with Bioconductor
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-4" class="callout-4-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Because of the complex and high dimensional nature of microbiome data, machine learning and other computational approaches have become an instrumental part of the researcher’s toolkit in this rapidly evolving field. There is an increasing need to develop robust and reproducible methods that take into account current and future trends in microbiome research such as multi-omics, expanding datasets, and longitudinal study designs. The previous solutions in microbiomics have fallen short in addressing these modern requirements, particularly in terms of scalability and data integration. To meet these challenges, the research community has extended the widely adopted SummarizedExperiment data container to TreeSummarizedExperiment, enabling support for microbiome-specific data structures such as hierarchical relationships. The framework is further enriched with mia (Microbiome Analysis) package family and packages by independent developers, providing methods for common data operations, visualization, and advanced analytical approaches. This session will showcase the latest advances in microbiome data science within Bioconductor, focusing on the mia (Microbiome Analysis) framework through a practical case study. We will also present Orchestrating Microbiome Analysis with Bioconductor, a freely available online book designed to promote best practices and facilitate adoption of the ecosystem. Together, these resources form a foundation for reproducible, scalable, and transparent microbiome data science, and they continue to evolve through active community contributions.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Christophe Vanderaa
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="msqrob2: robust modelling workflow for mass spectrometry-based proteomics" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-3-contents" aria-controls="callout-3" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
msqrob2: robust modelling workflow for mass spectrometry-based proteomics
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-3" class="callout-3-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Mass spectrometry (MS) has become a method of choice for exploring the proteome landscape that drives cellular functions. While technological advancements have significantly increased the sensitivity of MS instruments, obtaining reliable statistical results from these data remains a challenging and often tedious task. Many researchers continue to rely on ad-hoc analysis workflows due to a lack of clear guidelines, which can lead to violations of key statistical assumptions. In this workshop, we will offer a hands-on introduction to the msqrob2 package that provides a set of rigorously validated and benchmarked statistical workflows for MS-based proteomics. These workflows are built on the QFeatures framework for data processing. We will begin by familiarising participants with the input data format and the QFeatures data structure. From there, we will walk through the minimal data processing steps required prior to statistical modelling, explaining when and why each step is necessary. Next, we’ll explore the sources of variation inherent in proteomics data, highlighting their hierarchical structure and demonstrating how linear mixed models can properly account for these complexities. The modelling process will be carried out using msqrob2, which offers additional advantages such as robust and stabilised parameter estimation. Finally, we will demonstrate how to translate biological questions into hypothesis tests and how to prioritise proteomic markers that change in response to a condition of interest. Depending on the progress of the group, we will also briefly explore the emerging field of single-cell proteomics, discussing the additional challenges posed by these data. This workshop is designed for proteomics researchers who want to learn how to analyse their data using reproducible and statistically sound workflows, as well as for omics data analysts interested in expanding their skill set to include proteomics.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Alexandru Mahmoud
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="Navigating the New Bioconductor Workshop Platform: demonstrating how to author, wrap, and deploy your workshops" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-2-contents" aria-controls="callout-2" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Navigating the New Bioconductor Workshop Platform: demonstrating how to author, wrap, and deploy your workshops
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-2" class="callout-2-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Since the adoption of Galaxy as the underlying platform for serving Bioconductor Workshops, the process for authoring, wrapping, and deploying workshops has evolved. The recent direct collaboration effort with the Galaxy Project further introduces new changes in the automation available to workshop authors, while also bringing better integration and new avenues of interoperating with Galaxy. This workshop by a Bioconductor Core Team member aims to 1) introduce users to the easiest path for authoring material, empowering more users to create Bioconductor Workshops, and 2) introduce users to new features available in the Galaxy platform, especially ways in which Bioconductor users can now benefit from data persistence as well as interoperability with thousands of tools and workflows available from Galaxy. This workshop does not require any prior knowledge, and aims to be useful to users, current workshop authors, and prospective new authors alike.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Federico Marini
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
<div class="callout callout-style-simple callout-caution no-icon callout-titled" title="iSEE therefore I explore (better)" style="border-color: white; padding: 0rem; margin: 0rem;  font-family: Nunito Sans; font-weight: 100 !important;">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon no-icon"></i>
</div>
<div class="callout-title-container flex-fill">
iSEE therefore I explore (better)
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>iSEE (Interactive SummarizedExperiment Explorer) is a Bioconductor software package that provides a powerful and extendable multi-purpose visual interface for exploring data stored in a SummarizedExperiment object, using the R/shiny framework. Given the widespread adoption of SummarizedExperiment and its derivatives (SingleCellExperiment, SpatialExperiment, …) throughout the Bioconductor ecosystem and the smooth interoperability from other main frameworks (Seurat, Scanpy/AnnData), this package can be a ubiquitous companion across all the main steps of efficient analysis workflows, from the initial quality control all the way down to deploying data to accompany publications. In this workshop, we will provide an overview of its main functionality, displaying how the most common tasks in data exploration and interpretation can be achieved within this package, which delivers an efficient combination of interactivity and reproducibility. Attendees will be able to learn hands-on through a collection of vignettes that compose a masterclass-like full course on iSEE and its related packages (iSEEu, iSEEde, iSEEpathways, iSEEindex, and more). We aim to empower users in a “from zero to hero” format to explore in depth a wide spectrum of datasets, providing a free, natural, efficient, and customizable solution to achieve this within the Bioconductor project, and ultimately extract valuable insight from omics datasets in interdisciplinary settings.</p>
<div class="</td>">
</div></div></div></div></td></tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
16:30
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Poster session
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
<tr grouplength="5">
<td colspan="4" style="border-bottom: 1px solid #eee; ">
<strong>Fri. - Sep.&nbsp;19, ’25</strong>
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; background-color: lavender !important;padding-left: 2em;border-bottom: 1px solid lightyellow" indentlevel="1">
9:00
</td>
<td style="text-align:left;width: 20%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Keynote
</td>
<td style="text-align:left;width: 25%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Jacques Serizay
</td>
<td style="text-align:left;width: 40%; background-color: lavender !important;border-bottom: 1px solid lightyellow">
Enhancing genomic research with community-driven flexible software
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
9:30
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Flash talks
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; background-color: lightyellow !important;padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
10:30
</td>
<td style="text-align:left;width: 20%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
Break
</td>
<td style="text-align:left;width: 25%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; background-color: lightyellow !important;border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
11:15
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
BoF sessions
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
<tr>
<td style="text-align:left;width: 15%; padding-left: 2em;border-bottom: 1px solid lavender" indentlevel="1">
12:30
</td>
<td style="text-align:left;width: 20%; border-bottom: 1px solid lavender">
Awards &amp; Closing
</td>
<td style="text-align:left;width: 25%; border-bottom: 1px solid lavender">
Organizing committee
</td>
<td style="text-align:left;width: 40%; border-bottom: 1px solid lavender">
</td>
</tr>
</tbody>
</table>
<section id="posters" class="level2">
<h2 class="anchored" data-anchor-id="posters">Posters</h2>
<table class=" lightable-material" style="font-family: &quot;Source Sans Pro&quot;, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
author
</th>
<th style="text-align:left;">
title
</th>
</tr>
</thead>
<tbody>
<tr grouplength="26">
<td colspan="2" style="border-bottom: 1px solid #eee; ">
<strong>Wed. - Sep.&nbsp;17, ’25</strong>
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Sara Potente
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
SAMURAI: shallow analysis of copy number alterations using a reproducible and integrated bioinformatics pipeline
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Claire Rioualen
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Interoperability of Bioconductor packages within the ELIXIR Research Software Ecosystem using the EDAM ontology
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Edoardo Filippi
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Bringing Omics together with MOVIDA: MultiOmics Visualization, Integration and Downstream Analyses
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Ahmed Salah
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
DoReMiTra: A curated data package for radiation DOse REsponse Measured In TRAnscriptomics
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Kateřina Matějková
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
safRa: Splicing Annotation for Aberrant Fraction Analysis
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Angelo Velle
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
gINTomics, a powerful Bioconductor package for multiomics data integration and visualization.
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Dewy Nijhof
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Exploring Molecular Mechanisms of Comorbidity: A Network-Based Analysis of ADHD and Autism
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Alex Cecchetto
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Latent Structure Modeling in Spatial Transcriptomics Data
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Qiao Yang
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
BreastSubtypeR: Enhancing Reproducibility in Breast Cancer Research through Cohort-Adaptive Intrinsic Subtyping
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Jasper Spitzer
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
simpleHM: a heatmap visualisation package in the tidyverse
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Dany Mukesha
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Metabolomic Signatures and Machine Learning Models for Distinguishing Alzheimer’s Disease and Dementia with Lewy Bodies
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Antonino Zito
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
svdImpute2: An enhanced SVD-based imputation of randomly and non-randomly missing values in proteomics data
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Marta Sevilla Porras
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
UPDhmm: Detecting Uniparental Disomy through NGS Trio Data
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Sara Baldinelli
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Combinatorial Barcoding Meets In Vivo CRISPR: Decoding Context-Dependent Networks via sci-RNA-seq
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Aitor Moruno-Cuenca
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
singIST: an integrative method for comparative single-cell transcriptomics between disease models and humans
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Hiranyamaya Dash
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
MotifPeeker: R package for benchmarking epigenomic profiling methods using motif enrichment as a key metric
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Guillaume Deflandre
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
PSMatch: a Bioconductor Package for Handling Peptide-Spectrum Matches Data
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Stefania Pirrotta
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
mitology: a new tool to dissect mitochondrial activity from transcriptome
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Anna Bortolato
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Graph-based multi-omic survival and two-class pathway analysis with MOSClip R package
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Jiaqi Ni
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Nut consumption, gut microbiota, and cognitive function: findings from a prospective study in older adults at risk of cognitive decline
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Bernat Gel
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
scYoga: Identification of cellular stress in single-cell transcriptomics data
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Himanshu Saraswat
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Applying bioC packages to identify genes and pathways relevant in multi-case multiple sclerosis families.
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Artür Manukyan
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
VoltRon: A Spatial Omics Analysis Platform for Multi-Resolution and Multi-omics Integration using Image Registration
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Ali Mostafa Anwar
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
LimROTS: A Hybrid Method Integrating Empirical Bayes and Reproducibility-Optimized Statistics for Robust Differential Expression Analysis
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Anastasiia Horlova
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
A study of the genetic circuits of intestinal stem cell differentiation using in vivo CRISPR perturbation and single-cell RNA-seq
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Roger Olivella
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
ribomsqc: a Nextflow pipeline for automated QC of ribonucleoside MS data using MSnbase
</td>
</tr>
<tr grouplength="26">
<td colspan="2" style="border-bottom: 1px solid #eee; ">
<strong>Thu. - Sep.&nbsp;18, ’25</strong>
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Molka Anaghim Ftouhi
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
e-OMIX, a new visual interface for analyzing and managing omics data
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Laureano Tomas-Daza
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
HiCaptuRe: An R Package for Standardized Processing and Integration of Capture Hi-C Data
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Anastasiya Boersch
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
A Reproducible R Workflow for Flow Cytometry Data Analysis
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Charlotte Soneson
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
The Bioconductor ‘HowTo’ collection
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Laurent Gatto
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
An Open Software Development-based Ecosystem of R Packages for Mass Spectrometry Data Analysis
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Léopold Guyot
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Tools and Strategies for Systematic Benchmarking of R Packages: A Case Study with QFeatures
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Charlotte Soneson
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Eleven quick tips for writing a Bioconductor package
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Eliana Ibrahimi
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Fuzzy Forest for Microbiome-Driven Diagnosis of Cardiovascular Disease
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Andrea Mock
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Predicting therapy response in breast cancer patients using spatial proteomics of metastatic lymph nodes
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Alina Jenn
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
scBaptism: A unified tool for efficient and user-friendly single cell annotation
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Diego M. Fernandez-Aroca
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Epigenomic and transcriptomic changes associated to in vivo and in vitro cardiac hypertrophy
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Yosra Berrouayel Dahour
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Refining transcription factor enrichment analysis with the upgraded TFEA.ChIP
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Adrian Hernandez-Cacho
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Multi-omics approach identifies gut microbiota variations associated with depression.
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Kang Wang
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Extracting Tumor Cell Gene Expression Profiles and Metabolic Phenotypes Using PureMeta
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Elena Zuin
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Benchmark of single-cell batch correction methods available in the R and Python languages.
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Enes Sefa Ayar
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Benchmark of Module Detection Methods for Single Cell Proteomics
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Fjona Lami
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Explainable AI in Microbiome-Based Obesity Studies: From Black Boxes to Biological Insight
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Janet Piñero
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Streamlining Disease Genomics and Drug Discovery with disgenet2r
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Rike Hanssen
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Streamlining the Analysis Lifecycle: From Automated Processing to Interactive Analysis with Bioconductor, Nextflow, Seqera Containers, and Seqera Platform
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Axel Klenk
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
GSVA 2.x: Pathway-centric analysis at single-cell and spatial resolution
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Florian Auer
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Advancements of the RCX package adapting the Cytoscape Exchange format for biological networks
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Laura Masatti
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Decoding the Tumor Microenvironment Through Single-Cell Profiling
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Stefania Pirrotta
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
Public gene expression cancer signatures across bulk, single-cell and spatial transcriptomics data with signifinder
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Francesca A L Marino
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Defining an ADAR-Specific Transcriptional Signature to Infer Enzymatic Activity from RNA-Seq Data
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;" indentlevel="1">
Dario Righelli
</td>
<td style="text-align:left;font-size: small;width: 80%; ">
robin2: accelerating single cell data clustering evaluation
</td>
</tr>
<tr>
<td style="text-align:left;font-size: small;width: 20%; padding-left: 2em;background-color: rgba(242, 242, 242, 255) !important;" indentlevel="1">
Ivo Kwee
</td>
<td style="text-align:left;font-size: small;width: 80%; background-color: rgba(242, 242, 242, 255) !important;">
Combining multi-omics factorization methods for robust biomarker identification
</td>
</tr>
</tbody>
</table>


</section>
</section></main></div>





























































































































 <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
 <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p><a href="https://www.upf.edu/biomed"><img src="../images/logoUPF.png" class="img-fluid" style="width:30.0%"></a> <a href="https://www.isglobal.org"><img src="../images/logoISGLOBAL.png" class="img-fluid" style="width:30.0%"></a> <a href="https://prbb.org"><img src="../images/logoPRBB.png" class="img-fluid" style="width:30.0%"></a></p>
</div>   
    <div class="nav-footer-center">
<p>Copyright © 2025 Bioconductor <br><a href="mailto:eurobioc@bioconductor.org">eurobioc@bioconductor.org</a> <br><a href="https://www.bioconductor.org/about/code-of-conduct">Code of Conduct</a></p>
</div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/Bioconductor/EuroBioC2025">
      <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
    </a>
  </li>  
    <li class="nav-item">
    <a class="nav-link" href="https://bsky.app/profile/bioconductor.bsky.social">
<p><a href="https://bsky.app/profile/bioconductor.bsky.social"><img src="../images/logoBluesky.png" alt="Bluesky" height="15"></a></p>
</a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://genomic.social/@bioconductor">
      <i class="bi bi-mastodon" role="img" aria-label="Mastodon">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/company/bioconductor">
      <i class="bi bi-linkedin" role="img" aria-label="LinkedIn">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://slack.bioconductor.org">
      <i class="bi bi-slack" role="img" aria-label="Slack">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>




</body></html>